### PR TITLE
[Docenhancement][Addnote]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fa9375975c5bdcb2de203ecf405468cf824b80de/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c3a7ae7d-f8a2-4d08-8da6-a6f06424f0c3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-aad-sso.md](https://github.com/MicrosoftDocs/msteams-docs/blob/44aed023dd7adbbca75cf71e6a27031cc899aae7/msteams-platform/tabs/how-to/authentication/auth-aad-sso.md) | 96 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e2a4543-7e7a-460f-b5ff-2fd44e55c94e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved.

File: msteams-platform/tabs/how-to/authentication/auth-aad-sso.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ This section describes the tasks involved in creating a Teams tab that uses SSO.
 > * Only user-level Graph API permissions are supported that is, email, profile, offline_access, OpenId. If you must have access to other Graph scopes such as `User.Read` or `Mail.Read`, see [Get an access token with Graph permissions](#get-an-access-token-with-graph-permissions).
 > * It is important that your application's domain name is the same as the domain name you have registered for your AAD application.
 > * Currently multiple domains per app are not supported.
+> * The user must set the `accessTokenAcceptedVersion` to `v2` for a new application.
```
Comment (surbhigupta12) [inline msteams-platform/tabs/how-to/authentication/auth-aad-sso.md:65]: Please remove article "the"

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**